app/indexers/uri_to_string_behavior.rb

```diff
@@ -32,7 +32,8 @@ def uri_to_value_for(value)
 
     subject = RDF::URI.new(uri)
     predicate = RDF::URI.new(LABEL)
-    object = graph.query([subject, predicate, nil]).objects.first
+    objects = graph.query([subject, predicate, nil]).objects
+    object = objects.select { |o| o.language == :en }.first || objects.first
```

Comment:
For reference `#detect` or `#find` would be faster as it bails on the first object found.  But this is a small optimization.